README.md

```diff
@@ -1,24 +1,195 @@
+  <br/>
 
-* Database initialization
+</div>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- PROJECT DESCRIPTION. -->
 
-* Deployment instructions
+# 📖 Budget App <a name="about-project"></a>
 
-* ...
+> The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### <a name="tech-stack">
+Ruby on Rails
+</a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubygems.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org//">RoR</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Our project has the following features:
+
+- **Simplicity**
+- **Clarity**
+- **Accessibility**
+- **Responsiveness**
+- **Performance**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> As a developer you can clone the project and run it locally.
+
+To get a local copy up and running, follow these steps.
+
+1) Clone the repository
+2) cd into the project folder
+3) Run `bundle install` to install the required dependencies
+4) Run `rspec` to run the tests
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby on Rails
+- RSpec
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, copay the following URL:
+
+https://github.com/diegoh40/mybudget.git
+
+> Explanation of project.
+
+- [Video presentation.](https://www.loom.com/share/f74e52f4d5ae424fa6bd84db070d66c7)
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec
+```
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👥 **Author1**
+
+👤 Diego Hernández
+- GitHub: [@diehoh40](https://github.com/diegoh40)
+- Twitter: [@Diegoart80](https://twitter.com/Diegoart80)
+- LinkedIn:[ LinkedIn](https://www.linkedin.com/in/diegoarturoh/)
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Give a ⭐️ if you like this project!
+
+If you like this project...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) for this project, ` "The Creative Commons license of the design requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
" ` , please kindly acknowledge the author the original design, [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).